pkg/buildcontext/gc.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package buildcontext
+
+import "github.com/GoogleContainerTools/kaniko/pkg/util"
+
+// BuildContext unifies calls to download and unpack the build context.
+type GC struct {
```

Comment:
nit: this should probably be GCS instead of GC

Reply:
I know kops is using gc but I’ll check with the docs and such to make it match